src/main/webapp/i18n/de/textAssessment.json

```diff
@@ -39,7 +39,8 @@
                 "assess": "Bewerten"
             },
             "dashboard": {
-                "heading": "Einreichungen und Bewertungen der Textaufgabe \"{{exerciseTitle}}\""
+                "heading": "Einreichungen und Bewertungen der Textaufgabe \"{{exerciseTitle}}\"",
+                "progressDescription": "Fortschritt: {{ percentageAssessed }} % ({{ numberAssessed }} von {{ numberSubmissions }} eingereichten Abgaben bewertet)"
```

Comment:
Due to the refactor into a single component only the translation of the text exercises is used now, the translation keys in `programmingAssessment.json`, `modelingAssessment.json`, … are now unused.

Please remove the translations here and for the other exercise types and move this translation to the shared file `assessment.json`. It can be added under `artemisApp.assessment.dashboard` there.

The same applies to the `noSubmissions` key below as well.

Reply:
This is currently blocked by #5189 and was the reason why I started this PR. @Santia-go approached since he tried to switch between the different translations for every exercise types, but modeling exercises did not use the artemisApp-Prefix for keys, leading to an unnecessary complicated logic for getting the correct translation key. After my PR is merged, the different keys can get used again in a follow-up.